apps/condo/domains/user/access/AuthenticateOrRegisterUserWithTokenService.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Generated by `createservice user.AuthenticateOrRegisterUserWithTokenService --type mutations`
+ */
+
+/**
+ * Authorization and registration are available to any user
+ */
+async function canAuthenticateOrRegisterUserWithToken () {
+    return true
```

Comment:
What if deleted user tries?

Reply:
const userId = ctx.?authedItem.?id

// 
if (userId) {
   deleted
}

Reply:
I think this is a strange logic within the mutation of authorization registration. 
And if you write something like this, then the questions are about the logic of the mutation itself, and not about the accesses